sda-svc/templates/ingest-deploy.yaml

```diff
@@ -60,81 +60,148 @@ spec:
+        - name: BROKER_CLIENTKEY
+          value: {{ include "tlsPath" . }}/ingest.key
+        {{- end }}
+      {{- end }}
+        - name: BROKER_DURABLE
+          value: {{ .Values.global.broker.durable | quote }}
+        - name: BROKER_QUEUE
+          value: "files"
+        - name: BROKER_HOST
+          value: {{ required "A valid MQ host is required" .Values.global.broker.host | quote }}
+        - name: BROKER_PORT
+          value: {{ .Values.global.broker.port | quote }}
+        - name: BROKER_ROUTINGERROR
+          value: {{ .Values.global.broker.routingError }}
+        - name: BROKER_ROUTINGKEY
+          value: "archived"
+        - name: BROKER_VHOST
+          value: {{ .Values.global.broker.vhost | quote }}
+        - name: BROKER_SERVERNAME
+          value: {{ .Values.global.broker.host | quote }}
+        - name: BROKER_SSL
+          value: "true"
+        - name: BROKER_VERIFYPEER
           value: {{ .Values.global.broker.verifyPeer | quote }}
-        - name: INBOX_STORAGE_DRIVER
-      {{- if eq "S3Storage" .Values.global.inbox.storageType }}
-          value: S3Storage
-        - name: INBOX_S3_URL
-          value: {{ required "S3 inbox URL missing" .Values.global.inbox.s3Url }}
-        - name: INBOX_S3_BUCKET
+        - name: C4GH_FILEPATH
+          value: {{ template "c4ghPath" . }}/{{ default "c4gh.sec.pem" .Values.global.c4gh.file }}
+      {{- if or (eq "verify-ca" .Values.global.db.sslMode) (eq "verify-full" .Values.global.db.sslMode) }}
+        - name: DB_CACERT
+          value: {{ include "tlsPath" . }}/ca.crt
+      {{- end }}
+      {{- if eq "verify-full" .Values.global.db.sslMode }}
+        - name: DB_CLIENTCERT
+          value: {{ include "tlsPath" . }}/ingest.crt
+        - name: DB_CLIENTKEY
+          value: {{ include "tlsPath" . }}/ingest.key
+      {{- end }}
+        - name: DB_DATABASE
+          value: {{ default "lega" .Values.global.db.name | quote }}
+        - name: DB_HOST
+          value: {{ required "A valid DB host is required" .Values.global.db.host | quote }}
+        - name: DB_PORT
+          value: {{ .Values.global.db.port | quote }}
+        - name: DB_SSLMODE
+          value: {{ .Values.global.db.sslMode | quote }}
+      {{- if eq "s3" .Values.global.inbox.storageType }}
+        - name: INBOX_BUCKET
           value: {{ required "S3 inbox bucket missing" .Values.global.inbox.s3Bucket }}
-        - name: INBOX_S3_REGION
+      {{- if .Values.global.inbox.s3CaFile }}
+        - name: INBOX_CACERT
+          value: {{ template "tlsPath" . }}/{{ .Values.global.inbox.s3CaFile }}
+      {{- end }}
+        - name: INBOX_REGION
           value: {{ default "us-east-1" .Values.global.inbox.s3Region }}
+        - name: INBOX_TYPE
+          value: "s3"
+        - name: INBOX_URL
+          value: {{ required "S3 inbox URL missing" .Values.global.inbox.s3Url }}
       {{- else }}
-          value: "FileStorage"
+        - name: INBOX_TYPE
```

Comment:
```suggestion
```

Reply:
What about doing it like this? I personally don't like any of the previous solutions.

`{{- eq "s3" .Values.global.inbox.storageType | ternary "s3" "posix"  }}`

Reply:
Then you get an extra logic block outside of the `if` statement that configures `S3` or  `POSIX`